docs/usage.md

```diff
@@ -1,10 +1,16 @@
 # nf-core/eager: Usage
 
+## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/eager/usage](https://nf-co.re/eager/usage)
+
+> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
+
 ## Table of contents
 
 <!-- TOC depthfrom:2 depthto:3 -->
```

Comment:
Given that we now tell people not to read these markdown files on GitHub, and these ToC are not being rendered on the website, it might be best to remove them?